BookingsApi/BookingsApi.UnitTests/DAL/Queries/GetBookingsByCaseTypesQueryTests.cs

```diff
@@ -1,14 +1,20 @@
 using System;
+using BookingsApi.Common.Services;
+using BookingsApi.DAL;
 using BookingsApi.DAL.Queries;
 using BookingsApi.UnitTests.Utilities;
 using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Moq;
 using NUnit.Framework;
 // ReSharper disable ObjectCreationAsStatement
 
 namespace BookingsApi.UnitTests.DAL.Queries
 {
     public class GetBookingsByCaseTypesQueryTests : TestBase
     {
+        private Mock<IFeatureToggles> FeatureTogglesMock;
```

Comment:
There's a code smell for this being an unused variable. Should be able to revert all changes for this file

Reply:
done